src/wkwebview/mod.rs

```diff
@@ -347,6 +347,18 @@ impl InnerWebView {
         webview
       };
 
+      // disable background throttling if attributes.disable_background_throttling is true
+      // which works for iOS 17.0+,iPadOS 17.0+,Mac Catalyst 17.0+, macOS 14.0+, visionOS 1.0+
+      if attributes.disable_background_throttling {
```

Comment:
We may need to add a version check before setting the preference value in case an unsupported version will panic.

Reference:
https://github.com/tauri-apps/wry/blob/2ffcd494af480ddbade3d7568e4d133ba519d83f/src/wkwebview/mod.rs#L195

Reply:
Sometimes that's the case, sometimes not 😂 In this case i would assume it'll be ignored as well. I have a vm with something older than 14 that i could check tomorrow.

Reply:
And I think the attribute will affect iOS as well, so we might need to validate it on iOS, too 👀